proto/compiled/gogo_pb.d.ts

```diff
@@ -0,0 +1,160 @@
+// package: gogoproto
+// file: gogo.proto
+
+/* tslint:disable */
+/* eslint-disable */
+
+import * as jspb from "google-protobuf";
```

Comment:
Do we really need to commit the compiled code?

Reply:
It had crossed my mind but i didn't act on it. I have pushed an update, now the compilation is part of the `build` step, thanks for the nudge. :grin:

Reply:
@roman-mazur I see the compilation failed on the CI `spawn /tmp/build/66d12d10/versioned-source/node_modules/grpc-tools/bin/protoc ENOENT` any idea why the binary might be missing on the CI after npm install?

Reply:
I would understand if `ignore-scripts` argument was used but does not seem to be. :thinking:

Reply:
@codewithcheese it looks weird, but should not be something CI specific.
Are you sure you cannot reproduce it if you do a clean build locally like the CI does?

Reply:
I see the following logs related to grpc-tools installation:
```
> grpc-tools@1.9.0 install /tmp/build/323c6e89/versioned-source/node_modules/grpc-tools
> node-pre-gyp install

node-pre-gyp WARN Using request for node-pre-gyp https download 
[grpc-tools] Success: "/tmp/build/323c6e89/versioned-source/node_modules/grpc-tools/bin/grpc_tools.node" is installed via remote
```

How does it compare to your local machine output? Do you also have the `.node` file extension in the logs?

Reply:
Yeah I cannot reproduce it locally. I ran some more experiments and it looks like its an issue with `protoc` on alpine. https://www.flowdock.com/app/rulemotion/r-beginners/threads/7AQACCJLd5a1S5WcTv-0qiy-ocW

I have committed the compiled files again so that it passes CI other uses of the package dont also have issues.